scripts/get-real-world.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const request = require('request');
```

Comment:
How about we move to https://github.com/sindresorhus/got, which is a smaller dep that request?

Reply:
done